app/controllers/users.js

```diff
@@ -0,0 +1,6 @@
+const { createUser } = require('../services/users');
+
+exports.createUser = (req, res, next) =>
+  createUser(req.body)
+    .then(user => res.send({ name: user.name, last_name: user.last_name, email: user.email }))
```

Comment:
here you can do res.send({user}) or res.send(user), you avoid setting all properties by hand.
do you know the difference between those 2 ways?

Reply:
But, i don't want to send de password in the response, that's why im setting it like that

Reply:
Ohh I get it. In that case search about lodash pick function :)